home/templates/home/home_page.html

```diff
@@ -1,21 +1,13 @@
 {% extends "base.html" %}
-{% load static %}
+{% load wagtailcore_tags wagtailimages_tags %}
 
 {% block body_class %}template-homepage{% endblock %}
 
-{% block extra_css %}
-
-{% comment %}
-Delete the line below if you're just getting started and want to remove the welcome screen!
-{% endcomment %}
-<link rel="stylesheet" href="{% static 'css/welcome_page.css' %}">
-{% endblock extra_css %}
```

Comment:
The  `{% block extra_css %}` and `{% endblock extra_css %}` should probably stay, as this allows you to inject additional CSS to the page header specifically for this template
But the lines in between those should be removed :D